src/cmd_argument.rs

```diff
@@ -232,3 +232,33 @@ where
         config.working_directory = Some(self.0.as_ref().to_owned());
     }
 }
+
+/// Arguments of type [`PathBuf`] are passed into the child process
+/// as arguments.
+///
+/// ```
+/// use cradle::*;
+/// use std::path::PathBuf;
+///
+/// let current_dir: PathBuf = std::env::current_dir().unwrap();
+/// cmd_unit!("ls", current_dir);
+/// ```
+impl CmdArgument for PathBuf {
+    #[doc(hidden)]
+    fn prepare_config(self, config: &mut Config) {
+        config.arguments.push(self.into_os_string());
+    }
+}
+
+/// Arguments of type [`&Path`] are passed into the child process
+/// as arguments.
+///
+/// Similar to the impl for [`PathBuf`].
```

Comment:
Should this also have a doctest example?

Reply:
Yeah, maybe it should. It's really simple, but if the user happens to land on that doc item without knowing the library, a doctest example would be helpful.

Reply:
Done.